tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -376,6 +383,36 @@ func legacyNameRuleException(schemaName, providerName string) bool {
 	return schemaName == "runpod" && providerName == "runpod-native"
 }
 
+// legacyPublisherExceptions is a set of repository slugs that are exceptions to the
+// rule of requiring a publisher field in the schema.
+var legacyPublisherExceptions = codegen.NewStringSet(
+	"genesiscloud/pulumi-genesiscloud",
+	"nuage-studio/pulumi-nuage",
+	"wttech/pulumi-aem",
+	"pulumi/pulumi-tls-self-signed-cert",
+)
```

Comment:
It's great that the set of missing publishers is small.

Reply:
Yeah, I was thinking to send them PRs to just add the publisher